tests/adapter/dbt/tests/adapter/utils/test_equals.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from dbt.tests.adapter.utils.base_utils import macros__equals_sql
+from dbt.tests.adapter.utils.fixture_equals import (
+    SEEDS__DATA_EQUALS_CSV,
+    MODELS__EQUAL_VALUES_SQL,
+    MODELS__NOT_EQUAL_VALUES_SQL,
+)
+from dbt.tests.util import run_dbt, relation_from_name
+
+
+class BaseEquals:
+    @pytest.fixture(scope="class")
+    def macros(self):
+        return {
+            "equals.sql": macros__equals_sql,
+        }
+
+    # make it possible to dynamically update the macro call with a namespace
+    # (e.g.) 'dateadd', 'dbt.dateadd', 'dbt_utils.dateadd'
+    def macro_namespace(self):
+        return ""
+
+    def interpolate_macro_namespace(self, model_sql, macro_name):
```

Comment:
Help me understand the background of this step if ya please.

Reply:
Good question!

This is copy-pasta from [here](https://github.com/dbt-labs/dbt-core/pull/5428/files#diff-68f6cf649856a2177e399f35632114882e33d31d6378260d2ee73063ec6d52c8R5-R16).

I don't know all the details, but it's in a [couple places](https://github.com/search?q=repo%3Adbt-labs%2Fdbt-core+%22def+interpolate_macro_namespace%22&type=code) currently.

I don't remember what happens if I pull it out.

Reply:
I just tested removing it locally. It seems like the answer is "nothing" for this test at least 🤷‍♀️

Reply:
Cool -- thanks for checking. I pulled it out of this PR in 9aca97a00b60f4ea2af5afd546c6d54ff9a69a13.

We might want/need it to go back in if/when we do #6997 in order for it to be consistent with all the other cross-database macros [here](https://github.com/dbt-labs/dbt-core/tree/main/core/dbt/include/global_project/macros/utils) that inherit [this](https://github.com/dbt-labs/dbt-core/blob/dc35f56baa6876e6381b46f9b99b45f29d58f857/tests/adapter/dbt/tests/adapter/utils/base_utils.py#L28-L39).

But it might also be that we don't need it and the macro namespace interpolation logic was only needed for the migration of macros from dbt-utils to dbt-core 🤷